src/App.tsx

```diff
@@ -1,24 +1,59 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 0,
+  };
+
+  render() {
+    const { isClockVisible } = this.state;
+
+    return (
+      <>
+
+        <div className="App">
+          <h1>React clock</h1>
+          <p>
+            Current time:
+            {' '}
+            {isClockVisible ? <Clock name={this.state.clockName} /> : null}
```

Comment:
```suggestion
            {isClockVisible && <Clock name={this.state.clockName} />}
```
Можна використовувати conditional rendering без тернарників